content/en/opentelemetry/instrument/api_support/nodejs/metrics.md

```diff
@@ -0,0 +1,102 @@
+
+## Setup
+
+Follow these steps to enable OTel Metrics API support in your Node.js application.
+
+1. Install the Datadog SDK:
+   ```sh
+   npm install dd-trace
+   ```
+2. Enable OTel metrics by setting the following environment variable:
+   ```sh
+   export DD_METRICS_OTEL_ENABLED=true
+   ```
+4. Instrument your application:
+   ```javascript
+   [Code example]
+   ```
+   
+## Examples
+
+You can use the standard OpenTelemetry API packages to create custom metrics.
+
+### Create a counter
+
+This example uses the OTel Metrics API to create a counter that increments every time an item is processed:
+
+```javascript
+[Code example]
+```
+
+### Create a histogram
+
+This example uses the OTel Metrics API to create a histogram to track request durations:
+
+```javascript
+[Code example]
+```
+
+## Supported configuration
+
+To enable OTel Metrics support, you must set `DD_METRICS_OTEL_ENABLED=true`. The Datadog SDK also supports the following OpenTelemetry environment variables to customize your configuration:
+
+`DD_METRICS_OTEL_ENABLED`
+: **Description**: Enables the automatic instrumentation components of the Datadog SDK to collect and emit metrics generated by the OTel Metrics API. <br>
+**Default**: `false`
+
+`OTEL_EXPORTER_OTLP_ENDPOINT`
+: **Description**: Specifies the URL for sending OTLP data. <br>
+**Default (gRPC)**: `http://localhost:4317`. <br>
+**Default (HTTP)**: `http://localhost:4318/v1/metrics`. <br>
+
+`OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE`
+: **Description**: Specifies the aggregation `temporality` to use for each instrument kind. <br>
+**Accepted values**: `Cumulative`, `Delta`, `LowMemory`. <br>
+**Default**: `delta` <br>
+**Notes**: This default value is Datadog's recommended configuration and differs from the OpenTelemetry specification's default.
+
+For a complete list of all shared OTLP environment variables, see [OpenTelemetry Environment Variables Interoperability][1].
+
+### Tag and resource attribute precedence
+
+{{% otel-config-precedence %}}
+
+## Migrate from other setups
+
+### Existing OTel setup
+
+If you are already using the OpenTelemetry SDK with a manual OTLP exporter configuration, follow these steps to migrate:
+
+1. Add the Datadog SDK (`dd-trace`) to your project and enable its instrumentation.
+2. Remove any code that manually configures the `OTLPMetricsExporter`. The Datadog SDK handles this configuration automatically.
+3. Remove the `@opentelemetry/sdk-node` and `@opentelemetry/exporter-otlp` packages from your project's dependencies.
+4. Set the `DD_METRICS_OTEL_ENABLED=true` environment variable.
+
+### Existing DogStatsD setup
+
+If you are currently using the Datadog DogStatsD client and want to migrate to the OpenTelemetry Metrics API, you need to update your instrumentation code. The main difference is that OTel metrics are configured using environment variables rather than code, and you create `Instrument` objects first.
```

Comment:
We can link to opentelemetry nodejs onboarding docs here: https://github.com/open-telemetry/opentelemetry-js/blob/4dded45ec67f90998d75e3ddbde9872b6b466183/doc/metrics.md#getting-started